test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -711,8 +711,29 @@ namespace StackTest {
             $longPipeName = [string]::new("A", 200)
 
             "`$pid" | & $powershell -CustomPipeName $longPipeName -c -
-            # 64 is the ExitCode for BadCommandLineParameter
-            $LASTEXITCODE | Should -Be 64
+            $LASTEXITCODE | Should -Be $ExitCodeBadCommandLineParameter
+        }
+    }
+
+    Context "ApartmentState tests" {
```

Comment:
I think we need tests that STA really works where it is mandatory (COM?).

Reply:
I'll see if I can find a COM class to use that requires STA

Reply:
added